### DISCUSSION TITLE
img on sidebar/sidebar2

### DISCUSSION THREAD

**Discussion by mightygriffon6** [vikramlearning/blazorbootstrap#810] [Q&A] [answered]
Is there a way to put an image on sidebar first row
![image](https://github.com/user-attachments/assets/e38eab65-7326-4c51-bf2e-0a6f4623f08c)
I want an image instead of the title and the logo

**Comment by gvreddy04** [comment, marked as answer]
@mightygriffon6 t this moment, you can set an image as a logo only. We will support this feature in the upcoming release.

**Comment by gvreddy04** [comment]
https://github.com/vikramlearning/blazorbootstrap/issues/812
